doc/api/cli.md

```diff
@@ -625,6 +625,16 @@ Path to the file used to store the persistent REPL history. The default path is
 `~/.node_repl_history`, which is overridden by this variable. Setting the value
 to an empty string (`''` or `' '`) disables persistent REPL history.
 
+### `NODE_V8_COVERAGE=dir`
+
+When set, Node.js will begin outputting [V8 JavaScript code coverage][] to the
+directory provided as an argument. Coverage is output as an array of
+[ScriptCoverage][] objects.
```

Comment:
Just wondering, would an object with a single entry make sense? That’s leaves a bit of room for adding more information, e.g. metadata?

Reply:
the object is shaped like this:

```json
{
  "results": []
}
```